tests/openssh.rs

```diff
@@ -43,6 +43,32 @@ fn stdout() {
     session.close().unwrap();
 }
 
+#[test]
+#[cfg_attr(not(ci), ignore)]
+fn shell() {
+    let session = Session::connect(&addr(), KnownHosts::Accept).unwrap();
+
+    let child = session.shell("echo $USER").output().unwrap();
+    assert_eq!(child.stdout, b"test-user\n");
+
+    let child = session
+        .shell(r#"touch "$USER Documents""#)
+        .status()
+        .unwrap();
+    assert!(child.success());
+
```

Comment:
Might be nice to add a test of pipes, since that's the main reason I would use this. Maybe:
```suggestion
    let child = session.shell(r#"echo $USER | grep -c test"#).status().unwrap(); assert!(child.success());
```